--- origsrc/kplayer-0.7/doc/da/CMakeLists.txt	2008-01-11 07:27:10.000000000 -0600
+++ src/kplayer-0.7/doc/da/CMakeLists.txt	2009-07-13 23:35:48.637095400 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/da SUBDIR kplayer)
--- origsrc/kplayer-0.7/doc/en/CMakeLists.txt	2008-01-11 07:27:09.000000000 -0600
+++ src/kplayer-0.7/doc/en/CMakeLists.txt	2009-07-13 23:35:57.837236000 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/en SUBDIR kplayer)
--- origsrc/kplayer-0.7/doc/es/CMakeLists.txt	2008-01-11 07:27:11.000000000 -0600
+++ src/kplayer-0.7/doc/es/CMakeLists.txt	2009-07-13 23:36:03.452321800 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/es SUBDIR kplayer)
--- origsrc/kplayer-0.7/doc/it/CMakeLists.txt	2008-01-11 07:27:11.000000000 -0600
+++ src/kplayer-0.7/doc/it/CMakeLists.txt	2009-07-13 23:36:10.218675200 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it SUBDIR kplayer)
--- origsrc/kplayer-0.7/doc/pt/CMakeLists.txt	2008-01-11 07:27:10.000000000 -0600
+++ src/kplayer-0.7/doc/pt/CMakeLists.txt	2009-07-13 23:36:15.693758800 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt SUBDIR kplayer)
--- origsrc/kplayer-0.7/doc/sv/CMakeLists.txt	2008-01-11 07:27:10.000000000 -0600
+++ src/kplayer-0.7/doc/sv/CMakeLists.txt	2009-07-13 23:36:20.798836800 -0500
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR})
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv SUBDIR kplayer)
