diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x 'libtool.m4.in*' -x configure.bak -x configure.files -x acinclude.m4 -x subdirs -x configure.in origsrc/kplayer-0.6.3/kplayer/Makefile.am src/kplayer-0.6.3/kplayer/Makefile.am
--- origsrc/kplayer-0.6.3/kplayer/Makefile.am	2007-01-11 14:10:40.000000000 -0600
+++ src/kplayer-0.6.3/kplayer/Makefile.am	2007-07-29 19:55:56.734375000 -0500
@@ -45,7 +45,7 @@
 kde_module_LTLIBRARIES = libkplayerpart.la
 libkplayerpart_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN) -module -no-undefined
 libkplayerpart_la_SOURCES = kplayerpart.cpp
-libkplayerpart_la_LIBADD = $(LIB_KPARTS) $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT) libkplayernoinst.la
+libkplayerpart_la_LIBADD = libkplayernoinst.la $(LIB_KPARTS) $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT)
 libkplayerpart_la_METASOURCES = kplayerpart.moc
 
 # Uncomment the following two lines if you add a ui.rc file for your application to make use of KDEs XML GUI builing
