--- origsrc/kplayer-0.6.3/kplayer/Makefile.am	2007-01-11 14:10:40.000000000 -0600
+++ src/kplayer-0.6.3/kplayer/Makefile.am	2007-07-29 19:55:56.734375000 -0500
@@ -45,7 +45,7 @@
 kde_module_LTLIBRARIES = libkplayerpart.la
 libkplayerpart_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN) -module -no-undefined
 libkplayerpart_la_SOURCES = kplayerpart.cpp
-libkplayerpart_la_LIBADD = $(LIB_KPARTS) $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT) libkplayernoinst.la
+libkplayerpart_la_LIBADD = libkplayernoinst.la $(LIB_KPARTS) $(LIB_KFILE) $(LIB_KDEUI) $(LIB_KDECORE) $(LIB_QT)
 libkplayerpart_la_METASOURCES = kplayerpart.moc
 
 # Uncomment the following two lines if you add a ui.rc file for your application to make use of KDEs XML GUI builing
